css/styles.css

```diff
@@ -0,0 +1,85 @@
+* {
```

Comment:
why is there two css files? and why d they both set box-sizing: border-box;

Reply:
Fixed in 799c687